CHANGELOG.asciidoc

```diff
@@ -3,6 +3,442 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.7.0]]
+=== Beats version 7.7.0
+https://github.com/elastic/beats/compare/v7.6.2...v7.7.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Variable substitution from environment variables is not longer supported. {pull}15937{15937}
```

Comment:
for me is not clear, Do it means I can not use env var on beats config files?

Reply:
@blakerouse Can you help me understand the user impact here? Does this affect the documentation about using environment variables (note there is a section about specifying complex objects) in the config? See https://www.elastic.co/guide/en/beats/filebeat/7.7/using-environ-vars.html

Reply:
Yeah that works for me.